core/types/transaction.go

```diff
@@ -86,6 +87,7 @@ type TxData interface {
 	gasFeeCap() *big.Int
 	value() *big.Int
 	nonce() uint64
+	deadline() *big.Int
```

Comment:
this can be `uint64`

Reply:
are you sure, I started with uint64, but saw that the block_number is the same (*big.Int) type?